### ISSUE TITLE
from origin 'http://localhost:4200' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

### ISSUE THREAD

**Issue by andmaltes** [SignalR/SignalR#4392]
Hi, I am sorry that I know that this probably has been aswered a hundred of times but I can't make it work. It is suppoused to work like this but It is just showing the CORS error. 

I am using ASP FW 4.7

```
  public class Startup
    {
        public void Configuration(IAppBuilder app)
        {
        ...           
            app.Map("/signalr", map =>   {
        map.UseCors(CorsOptions.AllowAll);
            var hubConfiguration = new HubConfiguration
            {
                 EnableJSONP = true
            };
            map.RunSignalR(hubConfiguration);
        });             
        }
     
    }
```

Then in my angular APP:

```
  public initializeSignalRConnection(): void {
    const signalRServerEndPoint = 'http://localhost:52320/';
    this.connection = $.hubConnection(signalRServerEndPoint);
    this.proxy = this.connection.createHubProxy('NotificationHub');

    this.proxy.on('messageReceived', (serverMessage) => this.onMessageReceived(serverMessage));
    this.connection.start().done((data: any) => {
      console.log('Connected to Notification Hub');
      this.broadcastMessage();
    }).catch((error: any) => {
      console.log('Notification Hub error -> ' + error);
    });
  }

```
I am getting this 
![image](https://user-images.githubusercontent.com/38090766/60966943-095ce380-a2df-11e9-815f-6e9d0abe8ff0.png)

Thanks a lot

**Comment by anurse** [comment]
Can you provide a network trace using the Network tab of the browser dev tools? Instructions can be found here: https://docs.microsoft.com/en-us/aspnet/core/signalr/diagnostics?view=aspnetcore-2.2#collect-a-network-trace-in-the-browser (this documentation is for ASP.NET Core, but the instructions for creating a network trace apply to ASP.NET as well).

Please drag-and-drop the file in to the comment text box to "attach" it and *do not paste the file* as it is very large.

**Comment by anurse** [comment]
Closing this as we haven't heard from you and generally close issues with no response after some time. Please feel free to comment if you're able to get the information we're looking for and we can reopen the issue to investigate further!

**Comment by UzairAshraf1314141** [comment]
@anurse 
I'm facing the same problem 
Here is my HAR file. 
[localhost.zip](https://github.com/SignalR/SignalR/files/6838194/localhost.zip)

![image](https://user-images.githubusercontent.com/61478358/126107477-435ab6ef-1ad8-4819-9c6d-05df05f57483.png)
![image](https://user-images.githubusercontent.com/61478358/126107530-fc6d1981-0fd8-4aa4-9c40-cd424efb238f.png)

**Comment by anurse** [comment]
@UzairAshraf1314141 I'm actually no longer with the SignalR team, I suggest you open a new issue and someone on the team can look in to it!
